src/sagemaker/remote_function/client.py

```diff
@@ -82,49 +82,174 @@ def remote(
-            resources in a warm pool for subsequent training jobs. Default is 0.
-        max_retry_attempts (int): Max number of times the job is retried on InternalServerFailure.
-            Default is 1.
-        max_runtime_in_seconds (int): Timeout in seconds for training.  After this amount of time
-            Amazon SageMaker terminates the job regardless of its current status.
-            Default is 86400 seconds (1 day).
-        role (str): IAM role used for SageMaker execution.
-        s3_kms_key (str): The encryption key used for storing serialized data.
-        s3_root_uri (str): The root S3 folder where the code archives and data are uploaded to.
-        sagemaker_session (sagemaker.session.Session): The underlying SageMaker session which
-            AWS service calls are delegated to (default: None). If not provided, one is created
-            with default AWS configuration chain.
-        security_group_ids (List[str]): List of security group IDs.
-        subnets (List[str]): List of subnet IDs.
-        tags (List[Tuple[str, str]]): List of tags attached to the job.
-        volume_kms_key (str): KMS key used for encrypting EBS volume attached to the training
-            instance.
-        volume_size (int): Size in GB of the storage volume to use for storing input and output
-            data. Default is 30.
-        encrypt_inter_container_traffic (bool): Specifies whether traffic between training
-            containers is encrypted for the training job. (default: ``False``).
+          remote function. Only one of ``pre_execution_commands`` or ``pre_execution_script``
+          can be specified at the same time. Defaults to None.
+
+        environment_variables (Dict): The environment variables used inside the decorator function.
+          Defaults to ``None``.
+
+        image_uri (str): The universal resource identifier (URI) location of a Docker image on
+          Amazon Elastic Container Registry (ECR). Defaults to the following based on where the SDK
+          is running:
+
+            * For SageMaker Studio notebook cases, the image used as the kernel image for the
+              notebook is used.
+            * For other cases, it is resolved to base python image with the same python version
+              as the environment running the local code.
+
+          If no compatible image is found, a ValueError is thrown.
+
+        include_local_workdir (bool): A flag to indicate that the remote function should include
+          local directories. Set to ``True`` if the remote function code imports local modules and
+          methods that are not available via PyPI or conda. Default value is ``False``.
+
+        instance_count (int): The number of instance to use. Defaults to 1.
+
+        instance_type (str): The Amazon Elastic Compute Cloud (EC2) instance type to use to run
+          the SageMaker job. Valid values include ml.c4.xlarge. If not provided, ValueError is
```

Comment:
`Valid values include ml.c4.xlarge.`
It can be anything right?

Reply:
changed the statement to "e.g. ml..."